packages-next/types/src/utils.ts

```diff
@@ -4,7 +4,7 @@ export type BaseGeneratedListTypes = {
   backing: BackingTypeForItem;
   inputs: { create: GraphQLInput; update: GraphQLInput; where: GraphQLInput };
   args: {
-    listQuery: {
+    listQuery?: {
```

Comment:
Could you make this non-optional and for the printing case where we don't have args, instead of printing undefined, print `import("@keystone-next-types").BaseGeneratedListTypes["args"]["listQuery"]`?